engines/python/setup/djl_python/properties_manager/hf_properties.py

```diff
@@ -0,0 +1,218 @@
+import logging
+import os
+from enum import Enum
+from typing import Optional, Union
+
+import torch
+
+from pydantic import validator, root_validator
+
+from djl_python.properties_manager.properties import Properties, RollingBatchEnum
+
+
+class HFQuantizeMethods(str, Enum):
+    # added for backward compatibility lmi-dist
+    bitsandbytes = 'bitsandbytes'
+    gptq = 'gptq'
+
+    # huggingface
+    bitsandbytes4 = 'bitsandbytes4'
+    bitsandbytes8 = 'bitsandbytes8'
+
+    # supported by vllm
+    awq = 'awq'
```

Comment:
Kindly verify whether does it cover all supported quantization methods. lmi-dist has tests for gptq. vllm does not have test case for awq in our pipeline. Let me add it.

Reply:
there is no awq test for vLLM, feel free to add it